app/js/domains.json

```diff
@@ -1,4 +1 @@
-[
-  {"name": "gimmethe", "version": "1.0"},
-  {"name": "wantsome", "version": "1.0"}
-]
+{"name": "atpeepeth", "version": "1.0"}
```

Comment:
It looks like you're replacing the whole file with this line. Instead you need to add it into the array (as the first entry, for alphabetical ordering).